### PR TITLE
Force close server, even if there are open connections

### PR DESCRIPTION
**Pull request by krzkaczor**
Closes https://github.com/ethereumjs/testrpc/issues/365

### CODE REVIEW

**Comment by mattgstevens** [comment]
@krzkaczor think that the merge target needs to be `ethereumjs:develop` instead of `master` branch

**Comment by mattgstevens** [comment]
@krzkaczor ~~im going to open another PR that targets the correct branch since you havn't update the target branch in some weeks. would very much like to have this behaviour in master :)~~

im unsure of how things get merged in this project as the TravisCI will always error, and the develop branch is certainly stale.

@tcoulter any reason to not merge this in / how can we get this PR into an acceptable state?

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "url": "https://github.com/ethereumjs/testrpc"
   },
   "dependencies": {
+    "server-destroy": "^1.0.1",
```
Comment (moodysalem) [inline package.json]: i think you want this to be a dev dependency since the distribution is built with webpack

Comment (mattgstevens) [reply package.json]: good point, will make a new PR tomorrow if @krzkaczor continues to be quiet on this; it is quite annoying for me in dev workflow

**Comment by krzkaczor** [comment]
Hey guys @moodysalem @mattgstevens 
Sorry for delays - I am super busy with work and stuff. I was using my fork for some time and it for sure improves the situation but I noticed that meta mask still can behave weird. Like it's not re-establishing a new connection and fails silently. I am not sure if other ppl experience similar problems. I think we should leave a note on readme at least.

**Comment by mattgstevens** [comment]
@krzkaczor no worries; open source takes time :)

Think if you update this PR to move `server-destroy` into devDependencies as @moodysalem suggested it will be an easy yes to merge.

Not sure about the MetaMask issues you are mentioning, and seems like a separate issue. FWIW the dev environment I'm working in uses [concurrently](https://github.com/kimmobrunfeldt/concurrently) to start `testrpc` as one process and whatever you have for dev serving (webpack / express / ...) as another process. When the source is changing the watcher in the second process can do its thing, and the `testrpc` porcess stays alive.

**Comment by krzkaczor** [comment]
@mattgstevens  i am on it riight now. But I can't install project again with `yarn` because of some weird `node-gyp` problems. I think yarn 1.x broke it Oo

**Comment by krzkaczor** [comment]
@mattgstevens done. I rebased with master but it seems like its in corrupted state - doing yarn on master generates massive diff already. 🙄

**Comment by moodysalem** [comment]
that's a massive diff for adding a package that has no dependencies, maybe you should just start from master?

**Comment by krzkaczor** [comment]
@moodysalem the same happens on master - that's the thing. Just run `yarn` on master. It generates changes in few files.

**Comment by benjamincburns** [comment]
@krzkaczor the real problem was that we were committing generated build artifacts. I've since updated the `develop` branch to drop these. I think if you update this PR to drop these files from your commit, we'll be in good shape.

That said, I'm not well familiar with the dependency you're introducing here. What is it offering us? Should this be something which is actually handled in [trufflesuite/ganache-core](https://github.com/trufflesuite/ganache-core)?

**Comment by step21** [comment]
I would really like to see this. Is it ok to resubmit this, as I cannot update this PR obviously. I think server destroy is an ok dependency, it might also be possible to handle it in ganache-core but I do not really know ganache-core so not sure.

**Comment by szerintedmi** [comment]
this is a much waited pull request :) until it's merged I've found this workaround: https://github.com/trufflesuite/ganache-cli/issues/365#issuecomment-357098984

**Comment by benjamincburns** [comment]
I've fixed this in the `develop` branch of ganache-core.

**Pull request closed without merging**